java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -75,14 +83,42 @@
     if (nullCount.isPresent() && nullCount.get() > 0 && hostValidityBuffer == null) {
       throw new IllegalStateException("Buffer cannot have a nullCount without a validity buffer");
     }
-    if (type != DType.STRING) {
+    if (type != DType.STRING && type != DType.LIST) {
```

Comment:
Should we extend DType to include metadata about offsets?  This probably does not matter too much.